packages/nimble-components/src/wafer-map/modules/wafer-map-validator.ts

```diff
@@ -40,4 +42,25 @@ export class WaferMapValidator {
         }
         return !this.invalidGridDimensions;
     }
+
+    public validateDiesTableSchema(): boolean {
+        this.invalidDiesTableSchema = false;
+        if (this.wafermap.diesTable === undefined) {
+            this.invalidDiesTableSchema = false;
+        } else if (
+            this.wafermap.diesTable.numCols < 3
+            || this.wafermap.diesTable.schema.fields.findIndex(
```

Comment:
We should be checking the Type of the fields as well to align with what was in spec